packages/gmic/build.sh

```diff
@@ -16,7 +16,11 @@ termux_step_configure() {
 
 termux_step_make() {
 	cd src/
-	make cli
+	if [ $TERMUX_ARCH = "arm" ]; then
+		make cli  LIBS=" -lunwind -L/data/data/com.termux/files/usr/lib  -lfftw3 -lfftw3_threads -lcurl -lpng16 -lz -ljpeg -ltiff -lGraphicsMagick++ -lGraphicsMagick"
```

Comment:
I have to say this is no better than #2588, sorry.

Reply:
Its not meant to be better its the only way iv gotten gmic to run properly on arm. Using zlib from your pr didn't work even if built with -fexceptions.

Reply:
"Didn't work" doesn't really tell anything. You need to at least show that gmic is and is only linked to `libz.so.1` (or maybe the symlink `libtermuz.so`) but segfault still originates from it (the zlib). The former may not be the case if some hardcoded `-lz` is used at the end.

Reply:
It doesn't originate there its caused because the static libunwind.a isn't linked propertly.

Reply:
Not even sure what you mean. Actually there is even a note seeming to hint that we should avoid linking libraries against libgcc.a and libunwind.a (and now we may only need to exclude libgcc.a as it's a linker script to both the real libgcc and libunwind).

And still you are not showing any log (on what linkage the gmic has and how things have/have not changed with a zlib built by ourselves).

Reply:
Oh forgot the link: https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#Unwinding

Reply:
current gmic
Program received signal SIGSEGV, Segmentation fault.
0xf7189c3c in __gnu_Unwind_Resume () from /system/lib/libz.so
built on your zlib branch
Program received signal SIGSEGV, Segmentation fault.
0xf7407020 in __gnu_Unwind_Resume () from /system/lib/libm.so

its not linking properly since it looks kind of like this. 
https://github.com/android-ndk/ndk/issues/889

Reply:
Well you gmic was still linked against `/system/lib/libz.so`, so